test/unit/puppetlabs/services/ca/certificate_authority_core_test.clj

```diff
@@ -407,13 +416,23 @@
             (is (= #{localhost-status test-agent-status revoked-agent-status}
                  (set (json/parse-string (:body response) true))))))
 
+        (testing "returns json when no accept header specified"
+          (let [response (test-app
+                          {:uri "/v1/certificate_statuses/thisisirrelevant"
+                           :request-method :get
+                           :headers {}})]
+            (is (= 200 (:status response)))
+            (is (.startsWith (get-in response [:headers "Content-Type"]) "application/json"))
+            (is (= #{localhost-status test-agent-status revoked-agent-status}
+                   (set (json/parse-string (:body response) true))))))
+
         (testing "with 'Accept: pson'"
           (let [response (test-app
                           {:uri "/v1/certificate_statuses/thisisirrelevant"
                            :request-method :get
                            :headers {"accept" "pson"}})]
           (is (= 200 (:status response)))
-          (is (.startsWith (get-in response [:headers "Content-Type"]) "text/pson"))
+          (is (.startsWith (get-in response [:headers "Content-Type"]) "pson"))
```

Comment:
Why are we changing from `"text/pson"` to `"pson"`? Is this a change that's coming from ruby Puppet?